unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ProfileHUD/ProfileHUDViewV2.cs

```diff
@@ -199,10 +204,22 @@ public void ShowProfileIcon(bool show)
         profilePicObject.SetActive(show);
     }
 
-    public void ShowExpanded(bool show)
+    public void ShowExpanded(bool show, bool showMyAccountVersion = false)
```

Comment:
I completely agree with you on that we should implement a better approach for this, but taking into account that it would take some time to apply the proxy pattern refactor and that we should have this PR ready for today, I would prefer to move it out of this scope. Also, the old profile card is going to be removed as soon as this new feature live in production, we can re-think the design of this class once we are in that scenario.